evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -330,6 +330,43 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> CheatcodeStackExecutor<'a, 'b, B, P>
                     Token::FixedBytes(s_bytes.to_vec()),
                 ])]);
             }
+            HEVMCalls::Spoof(inner) => {
```

Comment:
Given these are new cheatcodes, could you please add a short description for them in the `CheatcodeHandler` struct doctests?

"Spoof executes an arbitrary contract call or transfer from an address. If an ETH transfer is also initiated via `{value: ...}`, it assumes that the specified caller has enough balance to execute the transfer. 

Example:

```
hevm.spoof{value: 1 ether}(address(1), address(2), abi.encodeWithSelector("transfer(address,uint256)", address(3), 4))
```

or something like that

"Deal `deal(address, uint256)` allows you to set an arbitrary account's balance to a specified value"

Reply:
i dont see existing docs anywhere. i added in the readme - i think in general since its exposed in solidity, it should be documented elsewhere anyways (like a readme)